vars/buildDotNetCore.groovy

```diff
@@ -42,9 +48,10 @@ void call(Map config=[:]) {
         config['validateClosure']()
       }
 
-
-      stage('Helm lint') {
-        test.lintHelm(repoName)
+      if(noHelm == 'false') {
```

Comment:
I'm having second thoughts about this variable name.  Because it's a negative it makes it difficult (for me anyway) to mentally process even this simple if statement.  

"if not helm is no then yes".

What if we reverse it and call it something more specific to the check.  Such as `hasHelmChart`.

If you use the `Boolean` data type you can then simplify this if statement to

`if (hasHelmChart)`

Which I think is easier to process